### PR TITLE
Update requirements django 3.2

### PR DESCRIPTION
**Pull request by submarcos**

### CODE REVIEW

**Review by submarcos — COMMENTED** [review]
set a changelog

**Review by numahell — COMMENTED** [review]
When I install from scratch in dev mode, I have some messages on `./manage.py migrate` for many models

```
zoning.RestrictedAreaType: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the ZoningConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
```

**Comment by submarcos** [comment]
> When I install from scratch in dev mode, I have some messages on `./manage.py migrate` for many models
> 
> ```
> zoning.RestrictedAreaType: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
> 	HINT: Configure the DEFAULT_AUTO_FIELD setting or the ZoningConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
> ```
pull again your branch https://github.com/GeotrekCE/Geotrek-admin/pull/3064/commits/d591b8d8b4268d30361f62879e963778b1c16ef6

File: requirements.txt [hunk 1]
```diff
@@ -55,72 +72,77 @@ coreschema==0.0.4
...
-datetime==4.3
+datetime==4.4
     # via appy
-defusedxml==0.6.0
+defusedxml==0.7.1
     # via cairosvg
-django==3.1.14
+deprecated==1.2.13
+    # via redis
+django==3.2.13
```
Comment (submarcos) [inline requirements.txt:87]:
```suggestion
django==3.2.15
```

File: docs/changelog.rst [hunk 2]
```diff
@@ -9,6 +9,18 @@ CHANGELOG
 
 - Disable debug log in debian package post installation script.
 
+**Maintenance**
+
+- Upgrade dependencies. The detail for the main dependencies:
+  + django to 3.2.13
```
Comment (submarcos) [inline docs/changelog.rst:15]:
```suggestion
  + django to 3.2.15
```

File: geotrek/maintenance/tests/test_views.py [hunk 3]
```diff
@@ -651,14 +651,14 @@ def setUpTestData(cls):
 
         cls.job1 = InterventionJobFactory(job="Worker", cost=12)
         cls.job2 = InterventionJobFactory(job="Streamer", cost=60)
-        cls.job1_column_name = "Cost Worker"
-        cls.job2_column_name = "Cost Streamer"
+        cls.job1_column_name = "Cost_Worker"
+        cls.job2_column_name = "Cost_Streamer"
```
Comment (Chatewgne) [inline geotrek/maintenance/tests/test_views.py:655]: Why adding an underscore ?

Comment (Chatewgne) [reply geotrek/maintenance/tests/test_views.py:655]: I see, thank you for the explanation :)

**Review by Chatewgne — APPROVED** [review]

File: geotrek/maintenance/tests/test_views.py (hunk 3 above)
Comment (marcantoinedupre) [reply geotrek/maintenance/tests/test_views.py:655]: Django 3.2.13 : https://docs.djangoproject.com/en/3.2/releases/3.2.13/

**Pull request merged**